app/api/private-cloud/products/[licencePlate]/comments/_operations/delete.ts

```diff
@@ -1,7 +1,8 @@
 import prisma from '@/core/prisma';
 
 export async function deleteOp(licencePlate: string, commentId: string) {
-  const deletedComment = await prisma.privateCloudComment.delete({
+  // Try to find and delete the comment in the context of a project
+  let deletedComment = await prisma.privateCloudComment.deleteMany({
```

Comment:
let's make it simple so to delete the comment by comment Id.
You will need to work on `/core/services` to have the security layer btw.